src/2981R0_improving_our_safety_with_a_physical_quantities_and_units_library.md

```diff
@@ -32,18 +34,21 @@ the library._
 
 # The Future Is Here
 
-Not that long ago, we watched SciFi movies about the future Earth where people were entering
-a car with no driver inside, and it autonomously drove them to the required destination.
-Then, it was something so futuristic and hard to imagine. Today, it slowly becomes a reality
-of our daily life.
+Not that long ago, self-driving cars were a thing from SciFi movies. It was something so
+futuristic and hard to imagine that it only appeared in movies set in the very far future.
+Today, autonomous cars are becoming a reality on our streets even if they are not (yet) widely
+adopted.
 
 It is no longer only the space industry or airline pilots that benefit from the autonomous
 operations of some machines. We live in a world where more and more ordinary people trust
-machines with their lives daily. Soon, we will be allowed to sleep while our
-car autonomously drives us home from a late party. As a result, many more C++ engineers are
-expected to write life-critical software today than a few years ago.
-Unfortunately, there is insufficient training and experience in this domain and the C++ language
-does not change fast enough to enforce a safe-by-construction code.
+machines with their lives daily. The autonmous car is just one example which will affect our 
+daily life. Medical devices such as surgical robots and smart health care devices are already
+a thing and will see wider adoption in the future. And there will be more machines with safety- 
+or even life-critical tasks in the future.
+As a result, many more C++ engineers are expected to write life-critical software today than a
+few years ago. Unfortunately, experience in this domain is hard to come by and training alone might
+not solve the issue of off-by-one-quantity mistakes. Additionally the C++ language does not change
```

Comment:
I would also probably like to add some sentences stating that the quantity equations are often not written by C++ experts, but by the common physics engineers who just happen to use C++ as a tool to code the logic they need.

Reply:
I mean things like using miles instead of km or passing seconds to a function instead of milliseconds or when a quantity is accidentally converted to another quantity through multiplication etc. i.e by the dev wanted to divide an length by a scalar to get a range increment but accidentally divides it by a time value and ends up with a speed (which might or might not be critical)
But I see that this needs a better explanation. It's also more off-by-one-unit mistakes